src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -19,13 +19,17 @@ class ElectronicDeliveryForm extends React.Component {
     this.state = {
       form: !_isEmpty(this.props.form) ? this.props.form :
         {
-          name: '',
-          email: '',
+          emailAddress: '',
```

Comment:
I'm not sure why all these were removed in the first place? When I tried to remove them it never worked for me. But it's probably not an issue now.